lib/logging/log.js

```diff
@@ -0,0 +1,415 @@
+var nodeutil = require('util');
+var pumpify = require('pumpify');
+var streamEvents = require('stream-events');
+var through = require('through2');
+
+/**
+ * @type {module:common/serviceObject}
+ * @private
+ */
+var ServiceObject = require('../common/service-object.js');
+
+/**
+ * A log is a named collection of records or entries, each entry representing a
+ * timestamped event. Logs can be produced by Google Cloud Platform services, by
+ * third-party services, or by your applications. For example, the log
+ * `apache-access` is produced by the Apache Web Server, but the log
+ * `compute.googleapis.com/activity_log` is produced by Google Compute Engine.
+ *
+ * @resource [Introduction to Logs]{@link https://cloud.google.com/logging/docs/api/#logs}
+ *
+ * @alias module:logging/log
+ * @constructor
+ *
+ * @param {object} options - [Configuration object](#/docs/?method=gcloud).
+ *
+ * @example
+ * var gcloud = require('gcloud')({
+ *   keyFilename: '/path/to/keyfile.json',
+ *   projectId: 'grape-spaceship-123'
+ * });
+ *
+ * var logging = gcloud.logging();
+ * var log = logging.log('my-log');
+ */
+function Log(logging, name) {
+  var methods = {
+    /**
+     * Delete the log.
+     *
+     * @resource [projects.logs.delete API Documentation]{@link https://cloud.google.com/logging/docs/api/ref/rest/v1beta3/projects.logs/delete}
+     *
+     * @param {function=} callback - The callback function.
+     * @param {?error} callback.err - An error returned while making this
+     *     request.
+     * @param {object} callback.apiResponse - The full API response.
+     *
+     * @example
+     * log.delete(function(err, apiResponse) {
+     *   if (!err) {
+     *     // The log was deleted.
+     *   }
+     * });
+     */
+    delete: true
+  };
+
+  this.name = name.replace(/^.*logs\//, '');
+  this.unformattedName = name;
+
+  ServiceObject.call(this, {
+    parent: logging,
+    baseUrl: '/logs',
+    id: this.name,
+    methods: methods
+  });
+}
+
+nodeutil.inherits(Log, ServiceObject);
+
+/**
+ * Return an array of log entries with the desired severity assigned.
+ *
+ * @private
+ *
+ * @param {object|object[]} entries - Log entries.
+ * @param {string} severity - The desired severity level.
+ */
+Log.assignSeverityToEntries_ = function(entries, severity) {
+  return arrify(entries).map(function(entry) {
+    return extend(true, {}, entry, {
+      metadata: {
+        severity: severity
+      }
+    });
+  });
+};
+
+/**
+ * Write a log entry with a severity of "ALERT".
+ *
+ * This is a simple wrapper around {module:logging/log#insert}. All arguments
+ * are the same as documented there.
+ *
+ * @example
+ * log.alert({
+ *   metadata: {
+ *     serviceName: 'compute.googleapis.com'
+ *   },
+ *   data: {
+ *     delegate: process.env.USER
+ *   }
+ * }, function(err, apiResponse) {});
+ */
+Log.prototype.alert = function(entry, labels, callback) {
+  this.insert(Log.assignSeverityToEntries_(entry, 'ALERT'), labels, callback);
```

Comment:
What I meant with the first question is: shouldn't the function call in example take 3 arguments?
(because alert() takes 3 arguments). 
I guess this works because I was the same in tests